main.sh

```diff
@@ -30,6 +30,53 @@ warn() {
 info() {
     echo "info: $*"
 }
+cargo_binstall() {
+    tool="$1"
+    version="$2"
+
+    target="$(rustc -vV | grep host | cut -c 7-)"
+    base_url=https://github.com/ryankurte/cargo-binstall/releases/latest/download/cargo-binstall
+    is_zip=false
+    case "${target}" in
+        x86_64-unknown-linux-gnu) url="${base_url}-x86_64-unknown-linux-musl.tgz" ;;
+        x86_64-unknown-linux-musl) url="${base_url}-x86_64-unknown-linux-musl.tgz" ;;
+
+        armv7-unknown-linux-gnueabihf) url="${base_url}-armv7-unknown-linux-musleabihf.tgz" ;;
+        armv7-unknown-linux-musleabihf) url="${base_url}-armv7-unknown-linux-musleabihf.tgz" ;;
+
+        aarch64-unknown-linux-gnu) url="${base_url}-aarch64-unknown-linux-musl.tgz" ;;
+        aarch64-unknown-linux-musl) url="${base_url}-aarch64-unknown-linux-musl.tgz" ;;
+
+        x86_64-apple-darwin | aarch64-apple-darwin | x86_64-pc-windows-msvc)
```

Comment:
Even if the host is windows-gnu, I think we can install the windows-msvc binary because it works on the GitHub Actions' environment.

Reply:
Thanks, I added a new branch here for target `x86_64-pc-windows-gnu`.